WorkflowSwiftUI/Sources/ActionModel.swift

```diff
@@ -0,0 +1,24 @@
+/// An ``ObservableModel`` for workflows with a single action.
+///
+/// Rather than creating this model directly, you should use the
+/// ``Workflow/RenderContext/makeActionModel(state:)`` method to create an instance of this model.
+public struct ActionModel<State: ObservableState, Action>: ObservableModel, SingleActionModel {
```

Comment:
nit: since `SingleActionModel` refines `ObservableModel` do we need both conformances here? i guess if it doesn't cause any compiler warnings it's fine, and maybe better to be more explicit. curious if there is a functional distinction with the `ObservableModel` conformance removed though

Reply:
No difference. `SingleActionModel` only exists so that it can be used as a constraint in some conveniences for when there's only one action.